Realm/RLMObjectSchema.mm

```diff
@@ -41,8 +41,8 @@ @interface RLMObjectSchema ()
 @implementation RLMObjectSchema {
     // table accessor optimization
     realm::TableRef _table;
-    NSArray *_propertiesInDeclaredOrder;
     NSArray *_swiftGenericProperties;
+    std::vector<RLMProperty *> _propertiesInTableColOrder;
```

Comment:
I'd suggest either `_propertiesInTableColumnOrder` or `_propertiesInTableOrder`. Seems a bit pointless to abbreviate to save 3 characters when the variable name is already > 25 characters long.